### PR TITLE
Adding locked state to ptable

### PR DESCRIPTION
**Pull request by sbernhard**
Requires:
- https://github.com/theforeman/foreman/pull/5741
- https://github.com/SatelliteQE/nailgun/pull/503

### CODE REVIEW

**Comment by sean797** [comment]
https://github.com/theforeman/foreman-ansible-modules/blob/8772e56c4d1fbe2133926bc106e93ffc667cf31c/modules/foreman_ptable.py#L36-L37 

will also need updating

File: modules/foreman_ptable.py [hunk 1]
```diff
@@ -351,7 +357,7 @@ def main():
         if state == 'present_with_defaults':
             module.fail_json(msg="'state: present_with_defaults' and 'name: *' cannot be used together")
         if state == 'absent':
-            if entity_dict.keys() != ['name']:
+            if entity_dict.keys() != ['name', 'locked']:
```
Comment (mdellweg) [inline modules/foreman_ptable.py]: What is the reasoning here?
I think, it breaks exactly, what the `if` was meant for.

Comment (akofink) [reply modules/foreman_ptable.py]: It doesn't make sense to provide `name` or `locked` if you're just removing the ptable (state = absent). Right?

Comment (akofink) [reply modules/foreman_ptable.py]: Oh, nevermind :) Yeah, this shouldn't be changed. It's saying that `name` is the *only* valid option when state = absent.

**Comment by sbernhard** [comment]
Fixed issue with default locked state, too. Similar to https://github.com/theforeman/foreman-ansible-modules/pull/139

**Comment by mdellweg** [comment]
Can you please add an entry in the test playbook to test this change.
It seems wrong that the tests pass without the necessary pending nailgun change.

**Comment by akofink** [comment]
Pretty stale here. Any updates, @sbernhard?

**Comment by mdellweg** [comment]
Seems, the nailgun patch has landed again.

**Comment by mdellweg** [comment]
A little bit of investigation lead me to the fact, that this depends on
https://github.com/SatelliteQE/nailgun/pull/542
The version requirement should therefore be nailgun > 0.31.0.

**Comment by akofink** [comment]
This label just means an associated Nailgun change was made, regardless of it's merged or not.

**Comment by akofink** [comment]
Thanks @sbernhard! And thanks to reviewers. Merging!

**Comment by mdellweg** [comment]
I thought it was meant like "Do not merge while this label shows, that a required nailgun change was pending."

**Pull request merged**